### PR TITLE
Update docs to include more detail about pathing.

### PR DESCRIPTION
**Pull request by Mineinjava**

### CODE REVIEW

File: pathing/Path.md [hunk 1]
```diff
@@ -8,13 +8,15 @@ icon: milestone
...
 ```
-If you don't know what a path is, swerve might not be for you
+If you don't know what a path is, swerve might not be for you.
 
 ## Usage
-You make an ArrayList of points (double[]), and you give it to the class. It represents those. Have fun :D
+You make an ArrayList<double[]>, and you pass it into the class. It represents the points that the robot will pass through, in the form [[_x_<sub>1</sub>, _y_<sub>1</sub>], [_x_<sub>2</sub>, _y_<sub>2</sub>], ... [_x_<sub>:icon-infinity:</sub>, _y_<sub>:icon-infinity:</sub>]]. Have fun :D
 
-Oh, you can cycle through the points using a few things, like `getCurrentPoint()`, `getNextPoint()`, and `getPointRelativeToCurrent()`.
+FinalHeading is now **required**. Just pass in the current gyro angle if you don't want the robot to rotate. Its also in radians because I'm not a monster.
```
Comment (cmontigel) [inline pathing/Path.md]: It's*

**Review by cmontigel — CHANGES_REQUESTED** [review]
LGTM other than your singular mistake, once that is fixed go ahead and merge. You're welcome for being annoying :D

**Comment by Mineinjava** [comment]
I cannot merge until you approve >>>:(

**Review by cmontigel — APPROVED** [review]
Hehe you know you love commit farming

**Pull request merged**